dashboard/app/models/plc/user_course_enrollment.rb

```diff
@@ -36,13 +36,15 @@ def self.enroll_users(user_emails, course_id)
     other_failure_users = []
     other_failure_errors = []
 
-    user_emails.each do |email|
-      user = User.find_by_email_or_hashed_email(email)
+    user_emails.each do |user_key|
+      user = user_key =~ /^\d+$/ ? User.find(user_key) : User.find_by_email_or_hashed_email(user_key)
```

Comment:
Please rename the param also to `user_keys` instead of `user_emails`, and add a function description with `@param` info